examples/networking-role/README.md

```diff
@@ -0,0 +1,3 @@
+# networking-roles
```

Comment:
I'm not a fan of this example because it doesn't teach them anything different about network-specific roles -- it's not a good example of what roles do in general because it doesn't use most of the features (no handlers, templates or files -- barely any vars and no defaults). Also a role with a single task file should be an include not role. 

Perhaps remove it so the rest of these can get in sooner?